### DISCUSSION TITLE
Object Detection using multirotor

### DISCUSSION THREAD

**Discussion by diya18114021** [microsoft/AirSim#4439] [Support Q&A]
After airsim build is completed, I want to detect object by running detection.py but it is giving me error (Exception Thrown). How to solve this issue ?
![Screenshot (53)](https://user-images.githubusercontent.com/45495906/160275593-7126d0bb-30c7-4f5e-a28d-25e44e9dfda4.png)

**Comment by zimmy87** [comment]
#4457 has now been merged, feel free to retest after updating to master.
